### DISCUSSION TITLE
Is it possible to use the reverse proxy in a local network?

### DISCUSSION THREAD

**Discussion by ghost** [NginxProxyManager/nginx-proxy-manager#2553] [Q&A]
I am trying to redirect my local home server IP so that I can access certain services hosted on certain ports using subdirectories.
```
192.168.1.10:8112
192.168.1.10/deluge
```
I tried setting my configuration like so but a `404 Not Found` page is returned when I try to access the `ip/subdirectory`:

![image](https://user-images.githubusercontent.com/63500907/213882576-6ac97b10-34b7-4d68-a151-8385d84a47c0.png)
![image](https://user-images.githubusercontent.com/63500907/213882550-0c9ff3de-b35f-4b79-a74f-9562ec7af460.png)

**Comment by norsemangrey** [comment]
Not sure if what you are trying to do there is possible, but if your router support it you can have an internal domain name (eg `*.at.home`) and you can create a host override alias `deluge.at.home` for your server IP in your DNS resolver and use the same in NPM. So you can use for instance `deluge.at.home` for `192.168.1.10:8112`. I do this for all my self hosted applications.

**Reply by RicTheRed** [reply]
I think I'm having a similar issue. What do you mean by "router support"? Does it have something to do with this issue:
https://www.reddit.com/r/unRAID/comments/xe30r5/cannot_reach_external_domains_locally_nginx_proxy/

Also to note, that I'm personally using Synology that has a built in nginx, but I decided to use one in Docker instead. I had to the following mapping instead which I'm not sure if that will effect access.

83:80, 84:81, and 444:443

Lastly I do not have these services opened to the external internet, but will eventually setup openvpn as the only way to access these services.
